README.md

```diff
@@ -1 +1,31 @@
-# flexy-thingworx-connector
\ No newline at end of file
+# Flexy Thingworx Connector
+
+The Flexy Thingworx Connector provides a connector-based solution development kit for linking the Ewon Flexy to a Thingworx instance.
+
+There are two components that make up the Flexy Thingworx Connector, a Thingworx service and a Flexy Java app.
```

Comment:
Use of app here is confusing **See other comment regarding this**

Reply:
fixed